spec/controllers/events_controller_spec.rb

```diff
@@ -1,5 +1,18 @@
 require 'rails_helper'
 
 RSpec.describe EventsController, type: :controller do
+  describe "#index" do
+    it "renders the index template" do
+      get :index
+      expect(response).to render_template(:index)
+    end
 
+    it "crawls for new meet up events using task job" do
+      Event.destroy_all
```

Comment:
We checked the previous commits, I believe Ethan was working on this /spec/controller